docs/accessibility/screen-readers.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Screen reader testing basics
+sidenavTitle: Screen reader basics
+permalink: /accessibility/screen-readers/index.html
+tags:
+  - accessibility
+order: 56
+---
+
+## General info
+
+This page provides some quick guidance for starting, stopping, and changing the settings of various screen readers. It also offers screen-reader-specific bonus tips and links to third-party resources like keyboard/gesture shortcut guides.
+
+### Why test with screen readers?
+
+When testing with screen readers, our goal is to verify that an assistive tech user will have an experience equivalent to that of any other user. Tests may include the following:
+
+1. Can you use and understand the purpose of all interactive elements on the page?
+2. Can you accomplish all the same tasks with and without assistive tech (e.g. navigating the site, submitting forms, adding items to a cart, etc.)?
+3. When page contents are read, are you presented with all the meaningful info you would expect as a visual user? (This includes status updates on dynamic pages.)
+
+### Browser and screen reader pairings
+
+If you can only test on one platform, it never hurts to try your screen reader with as many browsers as possible. For example, if you only have a Mac, go ahead and pair VoiceOver with Safari, Chrome, and Firefox.
+
+But if you’re either pressed for time or have access to multiple operating systems, these are the most common browser and screen reader pairings, and thus the ones to prioritize when testing:
+
+- macOS: Safari and VoiceOver
+- Windows: Firefox and NVDA
+- Windows: Chrome and JAWS or NVDA
+- Windows: Edge and Narrator
+- Linux: Firefox and Orca
+- iOS: Safari and VoiceOver
+- Android: Chrome and TalkBack
+
+## Computer screen readers
+
+On computers, screen reader testing often lets you perform two tests simultaneously. First, it ensures that all meaningful web content and interactions are available to assistive tech. And second, it helps you verify that your experiences are keyboard operable (which is a [WCAG requirement](https://www.w3.org/WAI/WCAG21/Understanding/keyboard-accessible.html)).
```

Comment:
Instead of "And second", maybe just "Second, ..." is better?

Also, "keyboard operable" should be hyphenated.